cmd/gardener-seed-admission-controller/app/gardener_seed_admission_controller.go

```diff
@@ -98,13 +99,16 @@ type Options struct {
 	Port int
 	// ServerCertDir is the path to server TLS cert and key.
 	ServerCertDir string
+	// AllowInvalidExtensionResources causes the seed-admission-controller to allow invalid extension resources.
+	AllowInvalidExtensionResources bool
 }
 
 // AddFlags adds gardener-seed-admission-controller's flags to the specified FlagSet.
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.BindAddress, "bind-address", "0.0.0.0", "address to bind to")
 	fs.IntVar(&o.Port, "port", 9443, "webhook server port")
 	fs.StringVar(&o.ServerCertDir, "tls-cert-dir", "", "directory with server TLS certificate and key (must contain a tls.crt and tls.key file)")
+	fs.BoolVar(&o.AllowInvalidExtensionResources, "allow-invalid-extension-resources", true, "Allow validation of extension resources")
```

Comment:
The default should be `false` (deny invalid extension resources).
```suggestion
	fs.BoolVar(&o.AllowInvalidExtensionResources, "allow-invalid-extension-resources", false, "Allow validation of extension resources")
```